node/network/collator-protocol/src/validator_side/mod.rs

```diff
@@ -960,6 +957,19 @@ where
 				"CollationFetchingRequest message is not expected on the validator side of the protocol",
 			);
 		}
+		Seconded(parent, stmt) => {
+			if let Some(collation_event) = state.seconded.remove(&parent) {
```

Comment:
the else branch means we seconded a collation, but the relay parent is deactivated, right, or it could be a race maybe?
probably worth logging the else branch with `debug!`